Some stub fixes

euresti opened the pull request:
Fixes a couple of issues found in the stubs.

Fixes #67

ilevkivskyi commented on sqlalchemy-stubs/sql/schema.pyi:
```diff
@@ -111,6 +117,12 @@ class Column(SchemaItem, ColumnClause[_T]):
                  server_onupdate: Union[FetchedValue, FunctionElement] = ..., quote: Optional[bool] = ..., unique: bool = ...,
                  system: bool = ..., comment: str = ...) -> None: ...
+    @overload
+    def __init__(self, name: str, *args: Any, autoincrement: Union[bool, str] = ...,
```
This overload is too lenient. Three comments here:
* Do you really need the `*args: Any`? This will make mypy accept calls like `Column('test', 42)` that fail at runtime.
* It would be great to infer `Column[None]` for this overload in the plugin (currently mypy will either infer `Column[Any]` or will ask for annotation).
* It is worth adding a tests for this (and maybe also for `ForeignKey` while you are at it).

euresti commented:
So I realized that my code was technically incorrect because it was not passing a type but I did not mean Column[NullType].  So I removed that overload and fixed my code.

I also wonder if I should make ForeignKey a Generic of _T.  What do you think?

I'll add the tests you suggested.

ilevkivskyi commented:
> I also wonder if I should make ForeignKey a Generic of _T. What do you think?

This is generally a good idea, but how the type argument will be inferred? It looks like this will need some help from the plugin. In any case this should be a separate PR.

> I'll add the tests you suggested.

OK, thanks, I will wait then.

euresti commented:
Ok. Test added!
